tool_path_planner/src/utilities.cpp

```diff
@@ -225,10 +230,20 @@ bool toPlaneSlicerConfig(PlaneSlicerRasterGenerator::Config& config,
 {
   config.point_spacing = config_msg.point_spacing;
   config.raster_spacing = config_msg.raster_spacing;
-  // config.tool_offset = config_msg.tool_offset;
   config.min_hole_size = config_msg.min_hole_size;
   config.min_segment_size = config_msg.min_segment_size;
   config.raster_rot_offset = config_msg.raster_rot_offset;
+  config.raster_wrt_global_axes = config_msg.raster_wrt_global_axes;
+
+  // Check that the raster direction was set; we are not interested in direction [0,0,0]
+  Eigen::Vector3d test_raster_direction;
+  test_raster_direction.x() = config_msg.raster_direction.x;
+  test_raster_direction.y() = config_msg.raster_direction.y;
+  test_raster_direction.z() = config_msg.raster_direction.z;
+  if (!test_raster_direction.isApprox(Eigen::Vector3d::Zero()))
+  {
+    config.raster_direction = test_raster_direction;
+  }
```

Comment:
Perhaps a warning should be thrown when this validation fails

Reply:
@marip8 and @jdlangs What are your thoughts? I think it may be better to throw an exception opposed to silently failing with a warning.

Reply:
If we assume that we intend to remove raster_wrt_global_axes and use the fact that norm(raster_direction) == 0 implies use default alignment method, and that norm(raster_direction)>0 means to use it, then this syntax is ok.